elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -174,47 +174,62 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
   ));
 }
 
-@include pfe-theme-contexts; // imports on="light" etc support
 
 /// ===========================================================================
 /// BELOW THIS POINT, LOCAL VARIABLE OVERRIDES ONLY, handled by the pfe-print-local function
 /// Maps can be found in the pfe-cta/src/_shared-assets.scss file
 
-// Broadcast colors: The CTA is tuning into whatever colors the containers are offering up, per priority level
-
-// - The location of these styles in the doc is important for clarity (readable code FTW!)
+// - The location of these styles in the doc is important for clarity and specficity 
 // - pfe-priority="primary" sets its own background colors, so it ignores broadcast
 // - if there is no broadcast variable available, these are still seen by the browser, so we also set fallbacks here!
 // - Fallback values should match whats above
 
+
+// The value of broadcasted variables is provided by the layouts so this seems uneeded?
+// @include pfe-theme-contexts; 
+
+
+:host([pfe-priority="primary"]) {
+  @include pfe-print-local($primary);
+}
+
 :host([pfe-priority="secondary"]) {
   @include pfe-print-local($secondary);
 }
 
+
 /// ===========================================================================
-/// PRIMARY CTAs
+/// CONTEXT CTA VARIANTS
 /// ===========================================================================
-:host([pfe-priority="primary"]) {
-  @include pfe-print-local($primary);
+
+
+:host([pfe-priority="primary"][on="dark"]),
+:host([pfe-priority="primary"][on="saturated"]) {
+  @include pfe-print-local($primary-saturated); 
+}
+
+:host([pfe-priority="secondary"][on="dark"]),
+:host([pfe-priority="secondary"][on="saturated"]) {
+  @include pfe-print-local($secondary-dark);
+}
+
+
+:host([pfe-priority="secondary"][on="dark"]),
```

Comment:
This appears to be duplicate to line 211

Reply:
ah yeah I think I intended to create a secondary-saturated set of vars but we didn't need it. Cleaned this up!